tests/test_benchmark_runner.py

```diff
@@ -317,3 +317,42 @@ def test_sut_caching(self, item_from_test, a_wrapped_test, tmp_path):
     # TODO: add stats
     # TODO: track errors
     # TODO: handle logs
+
+
+class TestRunTrackers:
+    def test_null(self, capsys):
```

Comment:
This is cool!

Reply:
Thanks! The capsys thing was new to me too. Nice to just have it handled.